ios/RNPurchases.m

```diff
@@ -191,11 +191,15 @@ - (dispatch_queue_t)methodQueue {
 
 RCT_EXPORT_METHOD(enableAdServicesAttributionTokenCollection)
 {
+    #if !TARGET_OS_TV
     if (@available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)) {
         [RCCommonFunctionality enableAdServicesAttributionTokenCollection];
     } else {
         NSLog(@"[Purchases] Warning: tried to enable AdServices attribution token collection, but it's only available on iOS 14.3 or greater or macOS 11.1 or greater.");
     }
+    #else
+    NSLog(@"[Purchases] Warning: AdServices attribution token collection is not available on tvOS.");
+    #endif
```

Comment:
I guess it's the only platform that doesn't have it
<img width="1119" alt="Screenshot 2024-03-22 at 11 08 11 AM" src="https://github.com/RevenueCat/react-native-purchases/assets/3922667/4d5e031c-27d4-4184-b22d-3fe7b9969c2e">